projects/novo-elements/src/elements/data-table/data-table.component.ts

```diff
@@ -65,13 +65,14 @@ import { DataTableState } from './state/data-table-state.service';
       <novo-data-table-pagination
         *ngIf="paginationOptions"
         [theme]="paginationOptions.theme"
-        [length]="dataSource?.currentTotal"
+        [length]="null !== overrideTotal ? overrideTotal : dataSource?.currentTotal"
```

Comment:
could this just be `overrideTotal ? overrideTotal : dataSource?.currentTotal` or did you specifically want to check for null?

Reply:
We wanted a null check because of the situation where overrideTotal=0, which is falsey, we still would want to use it.

Reply:
Ah good call. My main concern was around `undefined` which still might be worth checking
![Screen Shot 2022-09-09 at 12 16 34 PM](https://user-images.githubusercontent.com/7033445/189408029-1707f3c6-aa95-4a15-b4b2-478c61766db3.png)

Reply:
Good catch. I'll switch to overrideTotal >= 0